parity-clib/Parity.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace io.parity.ethereum;
```

Comment:
I don't know anything about Java, but I would expect users to copy-paste the file. Since we're not in a Gradle/Maven project, I don't think being in the right directory is relevant.

Reply:
Fair point about copy-paste, but the directory convention is not limited to Gradle/Maven, rather built into the java cli tooling. You can modify this default search path by `-classpath` though.